### DISCUSSION TITLE
set language option in backpack

### DISCUSSION THREAD

**Discussion by hetalvaghela106** [Laravel-Backpack/community-forum#827] [Q&A (Help)]
we want to use Hebrew language in locale we setup it in top header with flag. But whenever we are trying to change language content is remaining same. No effect of set locale in Hebrew. please help us to setup locale.
I already check thishttps://backpackforlaravel.com/docs/6.x/base-how-to#customize-the-translated-strings-aka-overwrite-the-language-file

**Comment by pxpm** [comment]
Hello @hetalvaghela106 

I think we don't have the Hebrew translations for Backpack. 😢 

Unfortunately none of our Hebrew speakers sent a PR with them, if you plan on doing them and submit a PR it would be highly appreciated. 🙏 

Currently we have this translations: https://github.com/Laravel-Backpack/CRUD/tree/main/src/resources/lang

Some are not complete, so if you want to start translating in Hebrew I recommend you start from the English as the base, as that one is for sure complete. 

Thanks in advance 🙏

**Comment by jcastroa87** [comment]
Due to not activity, I will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
